ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -167,17 +174,20 @@ private static String getServiceId(final Map<String, String> configs) throws Mig
 
     final Client ksqlClient = MigrationsUtil.getKsqlClient(
         ksqlServerUrl,
-        configs.get(KSQL_USERNAME),
-        configs.get(KSQL_PASSWORD),
+        configs.get(KSQL_BASIC_AUTH_USERNAME),
+        configs.get(KSQL_BASIC_AUTH_PASSWORD),
         configs.get(SSL_TRUSTSTORE_LOCATION),
         configs.get(SSL_TRUSTSTORE_PASSWORD),
         configs.get(SSL_KEYSTORE_LOCATION),
         configs.get(SSL_KEYSTORE_PASSWORD),
         configs.get(SSL_KEY_ALIAS),
         configs.get(SSL_KEY_PASSWORD),
-        configs.get(SSL_ALPN).equals("true")
+        configs.getOrDefault(SSL_ALPN, "false").equals("true"),
+        configs.getOrDefault(SSL_VERIFY_HOST, "true").equals("true")
     );
-
-    return ServerVersionUtil.getServerInfo(ksqlClient, ksqlServerUrl).getKsqlServiceId();
+    final String serviceId;
+    serviceId = ServerVersionUtil.getServerInfo(ksqlClient, ksqlServerUrl).getKsqlServiceId();
```

Comment:
This method could throw. We need to wrap this in a try-catch to ensure the client gets closed even if the server info call fails.